The other error is:

```
[ 86%] Linking CXX shared library libDllImportPath_Relative.so
/__w/1/s/src/tests/Interop/DllImportAttribute/DllImportPath/DllImportPathNative.cpp:13:24: error: 'main' should not be 'extern "C"' [-Werror,-Wmain]
   13 | extern "C" int __cdecl main(int argc,  char **argv)
      |                        ^
```

--- a/src/tests/Interop/DllImportAttribute/DllImportPath/DllImportPathNative.cpp
+++ b/src/tests/Interop/DllImportAttribute/DllImportPath/DllImportPathNative.cpp
@@ -10,7 +10,7 @@ extern "C" DLL_EXPORT int STDMETHODCALLTYPE GetZero()
 
 #ifdef EXE
 
-extern "C" int __cdecl main(int argc,  char **argv)
+int __cdecl main(int argc,  char **argv)
 {
     return 0;
 }

